### ISSUE TITLE
question about the test result

### ISSUE THREAD

**Issue by AGENT9717** [zeng-ziyin/PointNAT#1]
Firstly，I really appreciate your work, which solves the problem of information loss during local aggregation。

But when I run your code, the test miou is inconsistent with the paper, only 70.75, my running configuration is a piece of RTX4090, 12 vCPU Intel(R) Xeon(R) Platinum 8352V CPU @ 2.10GHz
seed select 1111 as the default,

I would like to know if this is the reason for my seed selection

Here is my run profile with the test results：
[s3dis-train-pointnat-ngpus1-20240912-204029-LLhrfNBfGcFPBJef8yVV6p.log](https://github.com/user-attachments/files/16987526/s3dis-train-pointnat-ngpus1-20240912-204029-LLhrfNBfGcFPBJef8yVV6p.log)

The key elements are as follows：
[09/12 20:40:29] S3DIS INFO: dist_url: tcp://localhost:8888
dist_backend: nccl
multiprocessing_distributed: False
ngpus_per_node: 1
world_size: 1
launcher: mp
local_rank: 0
use_gpu: True
seed: 1111
epoch: 0
epochs: 100
ignore_index: None
val_fn: validate
deterministic: False
sync_bn: False
criterion_args:
  NAME: CrossEntropy
  label_smoothing: 0.2
use_mask: False
grad_norm_clip: 10
layer_decay: 0
step_per_update: 1
start_epoch: 1
sched_on_epoch: True
wandb:
  use_wandb: False
  project: PointNeXt-S3DIS
  tags: ['s3dis', 'train', 'pointnat', 'ngpus1']
  name: s3dis-train-pointnat-ngpus1-20240912-204029-LLhrfNBfGcFPBJef8yVV6p
use_amp: False
use_voting: False
val_freq: 1
resume: False
test: False
finetune: False

09/13 09:09:29] S3DIS INFO: Best ckpt @E35,  val_oa 89.70, val_macc 75.56, val_miou 68.89, 
iou per cls is: [94.79 97.84 83.04  0.   35.01 56.79 74.99 79.6  89.64 70.61 74.57 76.71
 62.  ]
[09/13 09:09:30] S3DIS INFO: Successful Loading the ckpt from log/s3dis/s3dis-train-pointnat-ngpus1-20240912-204029-LLhrfNBfGcFPBJef8yVV6p/checkpoint/s3dis-train-pointnat-ngpus1-20240912-204029-LLhrfNBfGcFPBJef8yVV6p_ckpt_best.pth
[09/13 09:09:30] S3DIS INFO: ckpts @ 35 epoch( {'best_val': 68.8911361694336} )

iou per cls is: [ 94.4   96.9   69.83   0.     9.52   0.    87.16 100.     0.   100.
  72.49   0.    26.11]
[09/13 09:49:23] S3DIS INFO: Test [65]/[68] cloud
[09/13 09:49:39] S3DIS INFO: [65]/[68] cloud,  test_oa , test_macc, test_miou: 83.62 90.86 71.49, 
iou per cls is: [ 63.09  97.36  86.34 100.     0.     0.    44.92 100.   100.   100.
  78.62 100.    59.04]
[09/13 09:49:39] S3DIS INFO: Test [66]/[68] cloud
[09/13 09:49:40] S3DIS INFO: [66]/[68] cloud,  test_oa , test_macc, test_miou: 87.95 95.93 55.88, 
iou per cls is: [ 76.07  99.08  95.25 100.     0.     0.    95.08 100.     0.   100.
   0.     0.    60.98]
[09/13 09:49:40] S3DIS INFO: Test [67]/[68] cloud
[09/13 09:49:51] S3DIS INFO: [67]/[68] cloud,  test_oa , test_macc, test_miou: 86.36 85.34 60.79, 
iou per cls is: [ 94.65  98.64  91.94 100.   100.     0.    50.43  53.15  53.21 100.
   0.     0.    48.2 ]
[09/13 09:49:51] S3DIS INFO: Best ckpt @E35,  test_oa 90.78, test_macc 76.92, test_miou 70.75, 
iou per cls is: [95.59 98.55 85.01  0.   37.94 60.89 76.44 82.01 91.47 75.27 74.7  80.
 61.91]
[09/13 09:49:51] S3DIS INFO: save results in log/s3dis/s3dis-train-pointnat-ngpus1-20240912-204029-LLhrfNBfGcFPBJef8yVV6p/s3dis-train-pointnat-ngpus1-20240912-204029-LLhrfNBfGcFPBJef8yVV6p.csv

**Comment by yydslc** [comment]
My conclusion is the same as yours.
